src/services/fileServices/YmlFileServices/ConfigYmlService.js

```diff
@@ -17,7 +17,14 @@ class ConfigYmlService {
         fileName: CONFIG_FILE_NAME,
       }
     )
-    const content = sanitizedYamlParse(unparsedContent)
+    const sanitisedContent = sanitizedYamlParse(unparsedContent)
+    const sanitisedFacebookPixel =
+      !!sanitisedContent["facebook-pixel"] &&
+      sanitisedContent["facebook-pixel"].toString()
```

Comment:
thoughts on maintaining the invariant  (`[0-9]{15}`) either here (on read) or on write? if we do here on read, we must be sure that the `facebook-pixel` property is only ever gotten here.

Reply:
I think we should maintain the invariant on write, but not read! Reason for this is that users can still access github to modify it directly + they might have configured it wrongly in the past but never known - we don't want these to cause errors, and the user is informed via the FE settings not being able to save without modifying it

changed in https://github.com/isomerpages/isomercms-backend/pull/776/commits/5ad709e628452f372d0d982e070581892b016ccc